### PR TITLE
Fix #42

### LINKED ISSUE #42: less & scss support
less and scss doesn't work because it misses grammar. It should be cool to add those grammars and language configuration for thoses languages.

It should better too to consume VSCode css textmate https://github.com/Microsoft/vscode/blob/master/extensions/css/syntaxes/css.tmLanguage.json (to fix for CSS https://github.com/eclipse/tm4e/issues/135)

@jabby I think you could create a little PR for that (without your vue contribution), tell me if you have time, otherwise I will do it.

### PR DESCRIPTION
**Pull request by jabby**
The syntax coloring is ok but I don't understand how language configuration works. 

I add the needed files and the configuration in plugin.xml but it does not work as expected. Maybe @angelozerr can help for this point.

### CODE REVIEW

**Comment by angelozerr** [comment]
> but it does not work as expected

For the moment tm4e support language configuration only with auto close bracket and on enter support.

So if you type `{`, it should generate `}`, no?

Otherwise please create issues with  language configuration with missing features.

**Comment by jabby** [comment]
In fact it doesn't generate `}` I tried in css file and in json file. I will create an issue in tm4e project.

**Comment by angelozerr** [comment]
> In fact it doesn't generate } I tried in css file and in json file. I will create an issue in tm4e project.

Have you `org.eclipse.ui.genericeditor.autoEditStrategies` which is used at https://github.com/eclipse/tm4e/blob/master/org.eclipse.tm4e.languageconfiguration/plugin.xml#L21

**Comment by mickaelistria** [comment]
@angelozerr I let you work with @jabby on this one. Please drop a comment when you think this is ready to be merged.

File: org.eclipse.bluesky/language-configurations/less/language-configuration.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (angelozerr) [inline org.eclipse.bluesky/language-configurations/less/language-configuration.json]: Please add comments like https://github.com/mickaelistria/eclipse-bluesky/blob/master/org.eclipse.bluesky/language-configurations/css/language-configuration.json#L1

File: org.eclipse.bluesky/language-configurations/scss/language-configuration.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (angelozerr) [inline org.eclipse.bluesky/language-configurations/scss/language-configuration.json]: Please add comments like https://github.com/mickaelistria/eclipse-bluesky/blob/master/org.eclipse.bluesky/language-configurations/css/language-configuration.json#L1

File: org.eclipse.bluesky/language-configurations/less/language-configuration.json (hunk 1 above)
Comment (jabby) [reply org.eclipse.bluesky/language-configurations/less/language-configuration.json]: Hello I will fix this tomorrow.

File: org.eclipse.bluesky/language-configurations/scss/language-configuration.json (hunk 2 above)
Comment (jabby) [reply org.eclipse.bluesky/language-configurations/scss/language-configuration.json]: Hello I will fix this tomorrow.

**Comment by angelozerr** [comment]
@jabby I'm waiting for your fix to add color support for less.

@mickaelistria once @jabby will add license to JSON files, it's OK for me.

**Comment by jabby** [comment]
I updated the pull request. Fill free to comment / merge.

**Comment by angelozerr** [comment]
It's OK for me. Thanks @jabby !

**Comment by mickaelistria** [comment]
Please use a more descriptive commit message so someone looking at history would get what this change is about without looking at the code or the issue.

I suggest something like
```
Issue 42: Update TM Grammars and add Language Configurations

Signed-Off-By: Gautier de Saint Martin Lacaze <dark.jabberwock+github@gmail.com>
```
So it somehow conforms to Eclipse best practices if we move BlueSky to Eclipse.org one day.

**Comment by mickaelistria** [comment]
Rebased and merged as 9129818234766bbed93d6a8f2c9f134fc2fed74e . Thanks @jabby !

**Pull request closed without merging**